ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -64,9 +64,7 @@ periodics:
         value: "4.1"
       - name: TAG_SOURCE
         value: machine-os-content:$(BRANCH)
-      - name: CLUSTER_TYPE
-        value: aws
-      - name: CONFIG_SPEC
+      - name: UNRESOLVED_CONFIG
```

Comment:
You will also need to: 
1. remove the `--template` parameter
2. Remove the template volume mount
3. Remove the template volume